src/libs/actions/IOU.js

```diff
@@ -131,9 +131,10 @@ function startLoadingAndResetError() {
  * @param {String} currency
  * @param {String} recipientEmail
  * @param {String} debtorEmail
+ * @param {String} debtorName
  * @param {String} comment
  */
-function requestMoney(report, amount, currency, recipientEmail, debtorEmail, comment) {
+function requestMoney(report, amount, currency, recipientEmail, debtorEmail, debtorName = '', comment = '') {
```

Comment:
Not sure why we need default empty values here.

Reply:
I don't think they'll always be available. Mostly a sanity check

Reply:
This function is used only once and already state.comment is '' and fallback value for debatorName is also '' i.e if firstName is missing then we passed '' as default value in lodash

Reply:
That's fair. Removed.